fetch.bs

```diff
@@ -3981,13 +3981,17 @@ steps:
 <a for=/>response</a> <var>response</var>, run these steps:
 
 <ol>
+ <li><p>Let <var>timingInfo</var> be <var>response</var>'s <a for=response>timing info</a>.
+
  <li><p>Set <var>fetchParams</var>'s <a for="fetch params">request</a>'s
  <a for=request>done flag</a>.
 
+ <li><p>If <var>timingInfo</var> is null, then return.
```

Comment:
Why do we have this here and also below in finalize and report timing? It seems this doesn't need to be here and you should get process response done either way?